cli/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
 ## 13.6.1
 
+_Released 12/19/2023 (PENDING)_
+
+**Bugfixes:**
+
+- Release cy.origin logs between tests. Fixed in [#28465](https://github.com/cypress-io/cypress/pull/28465).
```

Comment:
@emilyrohrbough This isn’t super clear on what a user would see as the change to their tests from this description.

Reply:
better? https://github.com/cypress-io/cypress/pull/28465/commits/1ef9c4abf5625370b7eee8a588d63f5ca2098c30#diff-59059a9fb84116ac978d9f97f3138d4741854d92c1b44c2db71f7e64c23e3a01R8